New log file opened at Wed Dec 30 17:54:32 GMT 2015 by 5751@chrischivers-W65-67SR
Taking next connection to: 36213
New client attempting to connect on port: 52448-36213
  Address = /127.0.1.1:36213
This client on port 52448 has uuid 79f5abee-6e22-4bfa-b186-9e13d3f915d1 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: play-fork-run-79f5abee-6e22-4bfa-b186-9e13d3f915d1
Taking next connection to: 36213
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 36213
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 36213
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 36213
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 36213
New client attempting to connect on port: 53412-36213
  Address = /127.0.1.1:36213
Handshake exception on socket: 53412
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.io.EOFException
	at java.io.DataInputStream.readInt(DataInputStream.java:392)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 36213
New client attempting to connect on port: 53414-36213
  Address = /127.0.1.1:36213
This client on port 53414 has uuid c1d8821d-d277-48c7-99f4-47a59f7a6da6 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: play-fork-run-c1d8821d-d277-48c7-99f4-47a59f7a6da6
Taking next connection to: 36213
